recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
Yes, that is exactly the problem. How to get the full set of services running on the host.
The host /run is not visible in the containers, we neither bind- nor overlay-mount it.

One possibility I was considering is to generate container-specific preset-files based on the manifest. Roughly like this

container preset file generation:
disable *
foreach service in manifest do
  enable $service
done

Then every time (or only the first time) we boot the container we would do a 

systemctl --preset-mode=full preset-all

This would disable all services and enable only those that are explicitly listed in the manifest to be enabled.